tools/docker/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apt-get -qqy update \
     bzip2 \
```

Comment:
Unfortunately you can't change the docker file that easily; you'd need to update the version number and arrange for a manual push. However the fact that this is working suggests maybe dbus is already installed?

Reply:
Oh, so the changed dockerfile would not be used in the CI run? Do you know if it's possible to trigger wpt.fyi run for this PR using the updated Dockerfile? I want to test if this change makes the difference for the errors described in https://github.com/web-platform-tests/wpt/issues/42220.